src/org/zaproxy/zap/extension/frontendscanner/resources/Messages.properties

```diff
@@ -1,8 +1,9 @@
 # This file defines the default (English) variants of all of the internationalised messages
 
 frontendscanner.desc = Scan modern web applications
-frontendscanner.topmenu.tools.msg = Front-End Scanner is now {0}
-frontendscanner.topmenu.tools.title = Toggle Front-End Scanner
 
 frontendscanner.scripts.type.active = Client-side Active Rules
 frontendscanner.scripts.type.passive = Client-side Passive Rules
+
+frontendscanner.toolbar.button.on.tooltip = Enable the Front-end Scanner
```

Comment:
I'd suggest swapping the keys so that they match the button's state, easier to read in the code and for consistency with other add-ons.